sdk/compute/Microsoft.Azure.Management.Compute/tests/DiskRPTests/DiskRPTestsBase.cs

```diff
@@ -700,6 +1084,34 @@ private void Validate(DiskEncryptionSet diskEncryptionSetExpected, DiskEncryptio
             Assert.Equal(ResourceIdentityType.SystemAssigned.ToString(), diskEncryptionSetActual.Identity.Type);
```

Comment:
It would be good to validate the encryptionType here with the expected type

Reply:
added